packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -105,6 +124,25 @@ export class SDK implements ISDK {
       });
     }
 
+    if (options.endpoints) {
+      try {
+        this.endpoints = JSON.parse(options.endpoints);
+      } catch (_e) {}
```

Comment:
please align this with the style of the code in this file i.e. `catch (e)`

why swallow up the exception? should we be logging something here?

Reply:
Well, the whole idea is that it should just try, and if it fails it should swallow it and try the next one, with a final fallback at the end. 
I don't think it makes sense to log anything. Also i can align with the style, but that would make a warning of unused variable.

Reply:
I can add a debug log in the catch (e)?